src/gui/entry/EditEntryWidget.cpp

```diff
@@ -1014,8 +1014,36 @@ bool EditEntryWidget::commitEntry()
     }
 
     // Check Auto-Type validity early
-    if (!AutoType::verifyAutoTypeSyntax(m_autoTypeUi->sequenceEdit->text())) {
-        return false;
+    QString error;
+    if (m_autoTypeUi->customSequenceButton->isChecked()
+        && !AutoType::verifyAutoTypeSyntax(m_autoTypeUi->sequenceEdit->text(), m_entry, error)) {
+        auto res = MessageBox::question(this,
+                                        tr("Auto-Type Syntax Error"),
+                                        tr("The custom Auto-Type sequence failed validation:\n%1\n"
```

Comment:
Rephrase to sound less technical.

Reply:
done, used:
An error occurred while validating the custom Auto-Type sequence:
{Error Msg}
Would you like to correct it?